src/test/java/core/basesyntax/strategy/StrategyImplTest.java

```diff
@@ -0,0 +1,72 @@
+package core.basesyntax.strategy;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.handler.BalanceHandlerImpl;
+import core.basesyntax.strategy.handler.PurchaseHandlerImpl;
+import core.basesyntax.strategy.handler.ReturnHandlerImpl;
+import core.basesyntax.strategy.handler.SupplyHandlerImpl;
+import core.basesyntax.strategy.handler.TransactionHandler;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class StrategyImplTest {
+    private static Map<FruitTransaction.Operation, TransactionHandler> transactionHandlerMap;
```

Comment:
why make this a field?

Reply:
You men? i can create it like local variable?
Cause this map is necessary for strategyImpl constructor